packages/rum/src/boot/recorderApi.ts

```diff
@@ -155,29 +155,9 @@ export function makeRecorderApi(
 
         state = { status: RecorderStatus.Starting }
 
-        runOnReadyState(configuration, 'interactive', () => {
-          if (state.status !== RecorderStatus.Starting) {
-            return
-          }
-
-          const deflateEncoder = getOrCreateDeflateEncoder()
-          if (!deflateEncoder) {
-            state = {
-              status: RecorderStatus.Stopped,
-            }
-            return
-          }
-
-          const { stop: stopRecording } = startRecordingImpl(
-            lifeCycle,
-            configuration,
-            sessionManager,
-            viewHistory,
-            deflateEncoder
-          )
+        doStart().catch(() => {
```

Comment:
**💭 thought:** ‏It doesn't matter too much here, since the `catch` callback probably won't fail, but I think we'll need to `monitor()` Promise callbacks at the end of promise chains

Reply:
Done. Let me know what you think